app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -191,6 +192,21 @@ class Settings(
                 .apply()
         }
 
+    /**
+     This is needed for GUI Testing. If the value is not set in the sharePref
+     the default value will be the one from Nimbus.
+     */
```

Comment:
This block comment is missing some asterisks:

```suggestion
    /**
     * This is needed for GUI Testing. If the value is not set in the sharePref
     * the default value will be the one from Nimbus.
     */
```

Reply:
Done